cmd/activator/main.go

```diff
@@ -98,6 +99,12 @@ func main() {
 	}
 	createdLogger, atomicLevel := logging.NewLoggerFromConfig(config, logLevelKey)
 	logger = createdLogger.With(zap.String(logkey.ControllerType, "activator"))
+	if commmitID, err := changeset.Get(); err == nil {
+		// Enrich logs with GitHub commit ID.
+		logger = logger.With(zap.String(logkey.GitHubCommitID, commmitID))
+	} else {
+		logger.Warnf("Fetch GitHub commit ID from kodata failed: %v", err)
+	}
```

Comment:
Can we build this into `logging.NewLoggerFromConfig`, or whatever baseline logger that uses?

Reply:
The logging lib is from knative/pkg repository. We probably will not introduce dependency on knative/serving/pkg to knative/pkg/logging. An alternative is to add the changeset lib to knative/pkg/. But I feel this lib is something serving special. WDYT?

Reply:
Done